docs/source/installation.rst

```diff
@@ -6,19 +6,23 @@ Prerequisites
 
 - Start a server running latest Ubuntu version.
 
-- Enable password-less SSH access for :command:`root` user.
+- Enable password-less SSH access for :command:`ubuntu` user.
 
 - Partition and format any local disks you want to mount.
 
-- Verify the server's hostname matches the fully qualified domain name (FQDN).
-
 - Verify a valid DNS entry for the server.
 
 - Choose an SSL certificate source. Use either of these options:
 
   * `Let's Encrypt <https://letsencrypt.org/>`_
   * obtain a trusted SSL certificate and key for the server at that FQDN.
 
+- Checkout the latest version of the repository including the ``ansible-conda`` submodule::
+
+    $ git clone https://github.com/jupyterhub/jupyterhub-deploy-teaching.git
```

Comment:
The easiest way to do this is `git clone --recursive`, rather than three commands.